lib/generators/hyrax/work/work_generator.rb

```diff
@@ -1,70 +1,41 @@
 require 'rails/generators'
 
-class Rails::Generators::NamedBase
-  private
-
-    def destroy(what, *args)
-      log :destroy, what
-      argument = args.map(&:to_s).flatten.join(' ')
-
-      in_root do
-        run_ruby_script("bin/rails destroy #{what} #{argument}", verbose: true)
-      end
-    end
-end
-
 class Hyrax::WorkGenerator < Rails::Generators::NamedBase
   source_root File.expand_path('../templates', __FILE__)
 
   argument :attributes, type: :array, default: [], banner: 'field:type field:type'
 
   # Why all of these antics with defining individual methods?
   # Because I want the output of Hyrax::WorkGenerator to include all the processed files.
-  def create_model_spec
-    return unless rspec_installed?
-    template 'model_spec.rb.erb', "spec/models/#{file_name}_spec.rb"
-  end
-
-  def create_model
-    say_status("info", "GENERATING WORK MODEL", :blue)
-    template('model.rb.erb', "app/models/#{file_name}.rb")
-  end
-
-  def create_controller_spec
-    return unless rspec_installed?
-    template('controller_spec.rb.erb', "spec/controllers/hyrax/#{plural_file_name}_controller_spec.rb")
-  end
-
-  def create_actor_spec
-    return unless rspec_installed?
-    template('actor_spec.rb.erb', "spec/actors/hyrax/actors/#{file_name}_actor_spec.rb")
-  end
-
-  def create_form_spec
-    return unless rspec_installed?
-    template('form_spec.rb.erb', "spec/forms/hyrax/#{file_name}_form_spec.rb")
+  def banner
```

Comment:
Added this method so that the first step of the generator is displaying a status message.